odl/contrib/fom/test/test_supervised.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Tests for supervised FoMs."""
+
+import numpy as np
+import pytest
+import odl
+import odl.contrib.fom
+from odl.util.testutils import simple_fixture
+
+space = simple_fixture('space', [odl.rn(3),
+                                 odl.uniform_discr(0, 1, 10)])
+
+def test_mean_squared_error(space):
+    true = odl.phantom.white_noise(space)
+    data = odl.phantom.white_noise(space)
+
+    result = odl.contrib.fom.mean_squared_error(data, true)
+    expected = np.mean((true - data) ** 2)
+
+    assert pytest.approx(result) == expected
```

Comment:
Slightly weird to have it this way, I prefer `result == pytest.approx(expected)`